### PR TITLE
move distgit scl_enable here

### PR DESCRIPTION
**Pull request by akostadinov**
`scl_enable` was redundant between porta nad distgit repo. Lets use a single copy.

### CODE REVIEW

**Review by mayorova — DISMISSED** [review]
:+1:

**Comment by akostadinov** [comment]
I've tried the image in a test cluster and it works.

File: openshift/system/Dockerfile [hunk 1]
```diff
@@ -2,17 +2,24 @@ FROM quay.io/centos7/ruby-26-centos7
 
 USER root
 
+ENV RUBY_MAJOR_VERSION=2 \
+    RUBY_MINOR_VERSION=6
+
+ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}" \
+    RUBY_SCL_NAME_VERSION="${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}"
```
Comment (thalesmiguel) [inline openshift/system/Dockerfile:9]: Why do we need to ENV variables with the same value? Can't we just use `RUBY_VERSION`?

**Review by thalesmiguel — APPROVED** [review]
🚀

File: openshift/system/Dockerfile (hunk 1 above)
Comment (mayorova) [reply openshift/system/Dockerfile:9]: They are not the same :upside_down_face: 
One has a dot (`2.6`), and other does not (`26`)

**Pull request merged**